frontend/src/store/index.js

```diff
@@ -532,48 +532,50 @@ const getters = {
   projectNamesFromProjectList (state, getters) {
     return map(getters.projectList, 'metadata.name')
   },
-  customFieldsListShoot (state, getters) {
-    return map(getters.customFieldsShoot, (customFields, key) => {
+  shootCustomFieldList (state, getters) {
+    return map(getters.shootCustomFields, (customFields, key) => {
       return {
         ...customFields,
         key
       }
     })
   },
-  customFieldsShoot (state, getters) {
-    let customFieldsShoot = get(getters.projectFromProjectList, 'metadata.annotations["dashboard.gardener.cloud/customFieldsShoot"]')
-    if (!customFieldsShoot) {
-      return undefined
+  shootCustomFields (state, getters) {
+    function someObjectValuesAreObject () {
```

Comment:
Why is this a function factory? It has no parameters?

Reply:
✅